packages/babel-cli/src/babel/dir.js

```diff
@@ -84,8 +84,16 @@ export default async function({
 
   async function handleFile(src: string, base: string): Promise<boolean> {
     const written = await write(src, base);
-
-    if (!written && cliOptions.copyFiles) {
+    const relative = path.relative(base, src);
+    const isCompilableExtension = util.isCompilableExtension(
+      relative,
+      cliOptions.extensions,
+    );
+    if (
+      !written &&
+      ((!isCompilableExtension && cliOptions.copyFiles) ||
+        cliOptions.includeIgnored)
```

Comment:
I think when `copyFiles` is not enabled but `includeIgnored` is enabled, we should throw in `parseArgv`, otherwise we will have another flag that _could_ copy non compilable files.

Reply:
🚲🏠 warning:

Should we call it `--copy-ignored`? 😝

Reply:
Changed to --copy-ignored